src/components/FullScreenGallery.module.scss

```diff
@@ -46,7 +46,8 @@
         width: 100%;
         height: 100%;
         margin: auto;
-        object-fit: contain;
+        object-fit: cover;
```

Comment:
This causes a really big change to the tb.com gallery, especially on mobile. We might want to get input from Justin.

Before:
![Screenshot 2024-05-15 at 4 32 16 PM](https://github.com/tollbros/tollbrothers-ui/assets/15040013/0562d564-6d98-403e-82a6-b8558aee1c6d)


Your Branch:
![Screenshot 2024-05-15 at 4 32 09 PM](https://github.com/tollbros/tollbrothers-ui/assets/15040013/9e115b9a-6f41-4eec-987f-02563911a1a5)